### DISCUSSION TITLE
[PSD3] Exercise 5.35

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#133] [Esercizi (A-L)]
**Exercise 5.35** Convert the following two’s complement binary fixed-point numbers to base 10. The implied binary point is explicitly shown to aid in your interpretation.

(a) $0101.1000$
(b) $1111.1111$
(c) $1000.0000$

**Comment by Elia-Belli** [comment]
(a) $0101.1000_2 = 5.5_{10}$
(b) $1111.1111_2 = -0.0625_{10}$
(c) $1000.0000_2 = -8_{10}$

**Comment by FeddyLix17** [comment]
(a) corretto
(b) a me viene -1,9375
(c) corretto

**Comment by Elia-Belli** [comment]
@FedVlogger17 
(b) $1111.1111_2 = (-8+4+2+1+0,5+0.25+0.125+0.0625) = -0.0625_{10}$

**Comment by FeddyLix17** [comment]
@Elia-Belli si hai ragione avevo sbagliato i calcoli
